src/ansible_creator/arg_parser.py

```diff
@@ -290,6 +290,22 @@ def _add_resource_devfile(self: Parser, subparser: SubParser[ArgumentParser]) ->
             help="The destination directory for the devfile file. The default is the "
             "current working directory.",
         )
+        parser.add_argument(
```

Comment:
Rather than duplicate this here, I think we can pull it out into a seperate function, something like "add_overwrite" and call that fucntion from init and add passing in the parser.  It will eliminate the duplicate code and make it available to other subcommands in the future if needed.